src/api/app/models/bs_request_action_maintenance_incident.rb

```diff
@@ -124,20 +124,16 @@ def _merge_pkg_into_maintenance_incident(incidentProject)
                        project: linked_project, package: linked_package}
       ret = BranchPackage.new(branch_params).branch
       new_pkg = Package.get_by_project_and_name(ret[:data][:targetproject], ret[:data][:targetpackage])
-    else
-      # a new package for all targets
-      if linkinfo && linkinfo['package']
-        if Package.exists_by_project_and_name(incidentProject.name, source_package, follow_project_links: false)
-          new_pkg = Package.get_by_project_and_name(incidentProject.name, source_package, use_source: false, follow_project_links: false)
-        else
```

Comment:
To put the else and the if together, but all the code is still here. I just moved it : wink: